hw/chip/mimxrt10xx/family.c

```diff
@@ -329,22 +311,25 @@ void board_init(void)
     usb_phy->TX = phytx;
 
     //----------- FLEXSPI ----------//
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_06_FLEXSPI_A_SS0_B, 1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_07_FLEXSPI_A_DATA1,1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_08_FLEXSPI_A_DATA2, 1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_09_FLEXSPI_A_DATA0, 1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_10_FLEXSPI_A_SCLK, 1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_11_FLEXSPI_A_DATA3, 1U);
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_12_FLEXSPI_A_DQS, 1U);
-
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_06_FLEXSPI_A_SS0_B,0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_07_FLEXSPI_A_DATA1, 0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_08_FLEXSPI_A_DATA2, 0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_09_FLEXSPI_A_DATA0, 0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_10_FLEXSPI_A_SCLK, 0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_11_FLEXSPI_A_DATA3, 0x10E1U);
-    IOMUXC_SetPinConfig(IOMUXC_GPIO_SD_12_FLEXSPI_A_DQS, 0x10E1U);
-
+    IOMUXC_SetPinMux(PIN_SS0,   PIN_SS0_MUX);
+    IOMUXC_SetPinMux(PIN_DATA1, PIN_DATA1_MUX);
+    IOMUXC_SetPinMux(PIN_DATA2, PIN_DATA2_MUX);
+    IOMUXC_SetPinMux(PIN_DATA0, PIN_DATA0_MUX);
+    IOMUXC_SetPinMux(PIN_SCLK,  PIN_SCLK_MUX);
+    IOMUXC_SetPinMux(PIN_DATA3, PIN_DATA3_MUX);
+#ifdef PIN_DQS
+    IOMUXC_SetPinMux(PIN_DQS,   PIN_DQS_MUX);
+#endif
+    
+    IOMUXC_SetPinConfig(PIN_SS0,   PIN_SS0_CFG);
+    IOMUXC_SetPinConfig(PIN_DATA1, PIN_DATA1_CFG);
+    IOMUXC_SetPinConfig(PIN_DATA2, PIN_DATA2_CFG);
+    IOMUXC_SetPinConfig(PIN_DATA0, PIN_DATA0_CFG);
+    IOMUXC_SetPinConfig(PIN_SCLK,  PIN_SCLK_CFG);
+    IOMUXC_SetPinConfig(PIN_DATA3, PIN_DATA3_CFG);
+#ifdef PIN_DQS
+    IOMUXC_SetPinConfig(PIN_DQS,   PIN_DQS_CFG);
+#endif
```

Comment:
Please don't align the second argument like that, there's no point ;)

Reply:
Fixed.